tests/Native/AST.h

```diff
@@ -106,3 +106,13 @@ class Atomics
 # endif
 #endif
 };
+
```

Comment:
Thanks. I don't know why but I remembered from somewhere that functions have implicit and explicit declarations (not necessarily in class).
Well, I have fixed it now as you told. Please review.